src/main/java/bio/terra/workspace/service/iam/model/ControlledResourceIamRole.java

```diff
@@ -28,7 +28,7 @@
   public static ControlledResourceIamRole fromApiModel(ApiControlledResourceIamRole apiModel) {
     Optional<ControlledResourceIamRole> result =
         Arrays.stream(ControlledResourceIamRole.values())
-            .filter(x -> x.apiRole.equals(apiModel))
+            .filter(x -> x.apiRole == apiModel)
```

Comment:
prefer `.equals` over `==`, even for enums. go/java-practices/enums#equals
You can invert the order to avoid the NPE, but worth a comment that `x.apiRole` can be null. `apiModel.equals(x.apiRole)`

Reply:
done, with comment